web/modules/custom/collection/collection.behat.inc

```diff
@@ -288,9 +284,87 @@ class CollectionSubContext extends DrupalSubContextBase implements DrupalSubCont
     }
     // Also remove the image entities that were attached to the collections.
     foreach ($this->images as $image) {
-      file_unmanaged_delete($image->getFileUri());
       $image->delete();
     }
   }
 
+  /**
+   * Field alias mapping.
+   *
+   * @return array
+   *    Mapping.
+   */
+  private static function collectionFieldAliases() {
+    // Mapping alias - field name.
+    return [
+      'uri' => 'id',
+      'name' => 'label',
+      'abstract' => 'field_ar_abstract',
+      'access url' => 'field_ar_access_url',
+      'affiliates' => 'field_ar_affiliates',
+      'closed' => 'field_ar_closed',
+      'contact information' => 'field_ar_contact_information',
+      'creation date' => 'field_ar_creation_date',
+      'description' => 'field_ar_description',
+      'elibrary creation' => 'field_ar_elibrary_creation',
+      'logo' => 'field_ar_logo',
+      'moderation' => 'field_ar_moderation',
+      'modification date' => 'field_ar_modification_date',
+      'owner' => 'field_ar_owner',
+      'policy domain' => 'field_ar_policy_domain',
+      'spatial coverage' => 'field_ar_spatial_coverage',
+      'topic' => 'field_ar_topic',
+    ];
+  }
+
+  /**
+   * Assert that certain fields are present on the page.
+   *
+   * @param string $fields
+   *    Fields.
+   *
+   * @throws \Exception
+   *   Thrown when a column name is incorrect.
+   *
+   * @Then /^(?:|the following )fields should be present? "(?P<fields>[^"]*)"$/
+   */
+  public function assertFieldsPresent($fields) {
```

Comment:
This (and the `::assertFieldsNotPresent()` counterpart) is useful outside the scope of collections. I would move it to DrupalContext.